.github/workflows/Main.yml

```diff
@@ -408,6 +408,24 @@ jobs:
       run: |
         ./build/release/test/unittest --test-config test/configs/no_local_filesystem.json
 
+    - name: test/configs/small_block_size.json
+      if: (success() || failure()) && steps.build.conclusion == 'success'
+      shell: bash
+      run: |
+        ./build/release/test/unittest --test-config test/configs/small_block_size.json
+
+    - name: test/configs/default_storage_block_size_16kB.json
+      if: (success() || failure()) && steps.build.conclusion == 'success'
+      shell: bash
+      run: |
+        ./build/release/test/unittest --test-config test/configs/default_storage_block_size_16kB.json
```

Comment:
I think we can remove this - it is the same as `test/configs/small_block_size.json`, no?